api/api.go

```diff
@@ -70,10 +70,11 @@ func NewAPIWithVersion(ctx context.Context, globalConfig *conf.GlobalConfigurati
 	r.UseBypass(newStructuredLogger(logrus.StandardLogger()))
 	r.Use(recoverer)
 
+	r.Get("/", api.Index)
```

Comment:
This is already a route https://github.com/netlify/gocommerce/blob/659db7321413c22e22287c1aea5d9fe4e06267ff/api/api.go#L124

Reply:
Yeah - removing this again after updating kubernetes to use /health